test/integrate/proxywasm/proxywasm_wasmer_test.go

```diff
@@ -0,0 +1,17 @@
+//go:build wasmer
+// +build wasmer
+
```

Comment:
add
```
/*
 * Licensed to the Apache Software Foundation (ASF) under one or more
```

Reply:
plz make sure that every files contains those agreements.

Reply:
found one more.. I think that's it